packages/client/src/components/SafeSettings.js

```diff
@@ -349,6 +351,134 @@ const AddSafeOwner = ({ web3, onCancel, onNext, safeOwners }) => {
   );
 };
 
+const AddVault = ({ onCancel, onNext }) => {
+  const onNextClick = () => {
+    onNext({
+      coinType,
+    });
+  };
+
+  const nextButtonClasses = [
+    "button flex-1 p-4",
+    "is-link"
+  ];
+
+  const coinTypes = Object.entries(COIN_TYPE_TO_META).map((type) => ({
+    itemValue: type[0],
+    displayText: type[1].displayName,
+  }));
+
+  const [coinType, setCoinType] = useState(coinTypes[0].displayText);
```

Comment:
we should use itemvalue instead, they are the keys of the map.

Reply:
Why? They are the same values currently, is there a plan to change the display names in the future?

Reply:
Good question, the short answer is to keep the integrity of the coinType enum we defined. The pattern we practice here is "key(coinType)->attribute(displayName, vaultId...)", 
1. all the attributes (including the displayName) is subjected to change
2. all attributes should only be accessed by key
3. when passing arguments around we should avoid passing attributes and opt in passing by reference

There are a couple benefits
1. Single source of truth: displayName could be formatted/capitalized due to feature requirements and loose its integrity as a key to access other attributes of a coinType
2. Scalability: passing by reference allows the code that touches that argument not being impacted when we make changes to specific attributes